docker-compose.override.yml

```diff
@@ -30,13 +30,17 @@ services:
     build:
       context: .
       dockerfile: ./dev/django.Dockerfile
-    command: [
-      "celery",
-      "--app", "miqa.celery",
-      "worker",
-      "--loglevel", "INFO",
-      "--without-heartbeat"
-    ]
+    command:
+      [
+        "celery",
+        "-A",
+        "miqa.celery",
+        "worker",
+        "--loglevel",
+        "INFO",
+        "--without-heartbeat",
+        "-B" # add celery beat worker to schedule automatic tasks
```

Comment:
```suggestion
```

Rather than always running the `celery` container with `-B`, I'd suggest adding a totally new container (named something like `celerybeat`) with the exactly same other options as the `celery` container, except using:
```yaml
    command: [
      "celery",
      "--app", "miqa.celery",
      "beat",
      "--loglevel", "INFO"
    ]
```

That gives developers the ability (e.g. via `docker-compose up django celery`) to run only the Celery worker, without the beat service, which can sometimes be nice for debugging.